java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -124,42 +124,6 @@
     static final ResourceBundle rbx = ResourceBundle.getBundle("jmri.jmrit.display.DisplayBundle");
     static final ResourceBundle rbean = ResourceBundle.getBundle("jmri.NamedBeanBundle");
 
-    // size of point boxes
-    private static final double SIZE = 3.0;
-    private static final double SIZE2 = SIZE * 2.;  // must be twice SIZE
-
-    // connection types
-    final public static int NONE = 0;
```

Comment:
I assume you're talking about SIZE & SIZE2? or the connection types…
One of the reasons that I'm refactoring LayoutEditor.java (other than its HUGE size) is so that it won't need to know the connection types… Only the different track objects (LayoutTrack subclasses) will have to know about their own… Basically I want to move all the track specific code out of LayoutEditor.java.